spec/classes/foreman_proxy__proxydhcp__spec.rb

```diff
@@ -183,6 +183,16 @@
           }"
         end
 
+      context "on a non-existing interface" do
```

Comment:
The indenting looks inconsistent.

Reply:
I looked in Atom and it at spot 7 and the other contexts are as well

Reply:
It looks like you pasted in the middle of another context. That's why the indenting looks incorrect.